idd/Energy+.idd.in

```diff
@@ -28328,22 +28328,15 @@ HVACTemplate:Zone:WaterToAirHeatPump,
       \default 2.5
       \note The maximum on-off cycling rate for the compressor
       \note Suggested value is 2.5 for a typical heat pump
- N19, \field Heat Pump Time Constant
+ N19, \field Latent Capacity Time Constant
```

Comment:
IDD changes all appear to be functioning properly, and since they were generally copy/pasted, I don't feel inclined to scrutinize spelling and such.  Moving on...